ansible/group_vars/nimbus.sepolia.yml

```diff
@@ -108,6 +108,7 @@ nimbus_era_files_db_path: '/data/beacon-node-{{ beacon_node_network }}-unstable/
 nimbus_era_files_nclidb_path: '/data/beacon-node-{{ beacon_node_network }}-unstable/repo/build/ncli_db'
 
 # Open Ports
+open_ports_default_comment: 'Nimbus REST API'
```

Comment:
This will have no effect since all rules in `open_ports_list` have comments.

Reply:
Removed.